lambdas/WavWrapLambdaFunction/index.js

```diff
@@ -87,23 +87,13 @@ exports.handler = async (event) => {
   );
 
   const s3start = process.hrtime();
-  let s3Object = null;
-  try {
-    s3Object = await s3
-      .getObject({
-        Bucket: event.Artifact.BucketName,
-        Key: event.Artifact.ObjectKey,
-      })
-      .promise();
-  } catch (error) {
-    console.log(
-      'S3 getObject failed',
-      event.Artifact.BucketName,
-      event.Artifact.ObjectKey,
-      error,
-    );
-    throw error;
-  }
+
+  const s3Object = await s3
+    .getObject({
+      Bucket: event.Artifact.BucketName,
+      Key: event.Artifact.ObjectKey,
+    })
+    .promise();
```

Comment:
A `getObject` error will throw its own error, and the event is already being logged, so this try/catch was duplicative.